Initial

zacblazic opened the pull request:
A basic reverse proxy designed to simplify header manipulation.

zacblazic committed:
Add Dockerfile

zacblazic committed:
Add .dockerignore

zacblazic committed:
Add docker-compose.yml

zacblazic committed:
Add configuration files

zacblazic committed:
Add CHANGELOG

zacblazic committed:
Add headers-more nginx module

This module gives us more control over header manipulation, such as
being able to remove headers.

zacblazic committed:
Load headers-more module

zacblazic committed:
Remove server header

zacblazic committed:
Improve example

zacblazic committed:
Sort alpine dependencies

itskingori commented on CHANGELOG.md:
```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## 1.0.0
```
Shouldn't this follow the Nginx version i.e. `1.19.5`?

zacblazic replied:
We can do that, I'll update.

zacblazic replied:
Done in https://github.com/Intellection/docker-nginx-proxy/pull/1/commits/54d52fc5d58c8e6171abf7a5681cf401910042d4.

zacblazic committed:
Split configuration into files

To enable customisation of things like logging and settings in
the main context.

zacblazic committed:
Update README

zacblazic committed:
Track nginx version
